sdk/cognitiveservices/arm-cognitiveservices/src/cognitiveServicesManagementClientContext.ts

```diff
@@ -13,7 +13,7 @@ import * as msRest from "@azure/ms-rest-js";
 import * as msRestAzure from "@azure/ms-rest-azure-js";
 
 const packageName = "@azure/arm-cognitiveservices";
-const packageVersion = "5.1.0";
+const packageVersion = "5.2.0";
```

Comment:
Why is `@azure/arm-cognitiveservices` being touched in this PR which is meant for `@azure/arm-mediaservices`?